### PR TITLE
feat(@vtmn/svelte): create a shadow directive

### PR DESCRIPTION
**Pull request by Tlahey**
The main goal of this PR is to create a directive in svelte allowing to manage dynamically the shadows.
The goal is to be able to dynamically change the shadow of an element thanks to a reactive variable.

Todo : 
- [x] Create a directive on the guideline directory using the vtmn shadow classes
- [x] Change the `rollup.conf` to includes guideline dir
- [x] Add a storybook to test the directive

Extra : 
- **Need to be discute** : remove the `import '@vtmn/css-design-tokens';` on the `packages/sources/svelte/src/index.js`file. he seems to not change the component. And with SSR, he can cause problems.

### CODE REVIEW

**Review by GaspardMathon — APPROVED** [review]

File: packages/showcases/svelte/stories/guidelines/shadows/VtmnShadows.stories.svelte [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      </span>
+    </div>
+  </div>
+</Template>
+
+<Story name="Overview" />
+
+<style>
+  @import '@vtmn/css-tailwind-preset';
+</style>
```
Comment (lauthieb) [inline packages/showcases/svelte/stories/guidelines/shadows/VtmnShadows.stories.svelte:41]: Why? I don't understand why you import a preset.

File: packages/sources/svelte/rollup.config.js [hunk 2]
```diff
@@ -36,4 +42,15 @@ export default components.map(({ folder, name }) => ({
...
+export default [
+  ...components.map(({ folder, name }) => ({
+    input: `src/components/${folder}/${name}/${name}.svelte`,
+    ...svelteOptions(name),
+  })),
+  ...guidelines.map(({ folder, name }) => ({
+    input: `src/guidelines/${folder}/${name}.js`,
+    ...svelteOptions(name),
+  })),
+];
```
Comment (lauthieb) [inline packages/sources/svelte/rollup.config.js:56]: Nice, thanks!

**Review by lauthieb — CHANGES_REQUESTED** [review]
Thanks! just few question

**Comment by lauthieb** [comment]
Hi @Tlahey, thanks a lot! 
As I have some question, I propose you to make draft again this PR. 
Next week I will be on vacation but feel free to answer me, I'll check asynchronously. :)

File: packages/showcases/svelte/stories/guidelines/shadows/VtmnShadows.stories.svelte (hunk 1 above)
Comment (Tlahey) [reply packages/showcases/svelte/stories/guidelines/shadows/VtmnShadows.stories.svelte:41]: My bad, I will remove this line !

**Review by GaspardMathon — APPROVED** [review]

**Comment by lauthieb** [comment]
Hi @Tlahey, can you please fix conflicts? Thx a lot!

**Review by lauthieb — COMMENTED** [review]
Hi @Tlahey can you please fix conflits? Thx a lot!

**Review by lauthieb — CHANGES_REQUESTED** [review]
Hi @Tlahey can you please fix conflits? Thx a lot!

**Comment by lauthieb** [comment]
Hi @Tlahey,
First of all, thanks a lot for your work and this proposition.
Unfortunately, I will not accept it because I find that it adds a syntactic sugar that is not very useful since the CSS class is already quite semantic and easy to use in a Svelte application.
So I close this Pull Request, of course, I stay tuned if needed via an issue or a discussion on GitHub.
Thanks for your contribution and the efforts given for Vitamin Antoine. 💌

**Pull request closed without merging**